### PR TITLE
Added FAQ section about backing up root partition

### PR DESCRIPTION
**Pull request by fmikker**
Added a small note in the FAQ about backing up the root partition.

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -353,6 +353,10 @@ How can I restore huge file(s) over a unstable connection?
 If you can not manage to extract the whole big file in one go, you can extract
 all the part files (see above) and manually concatenate them together.
 
+How can I backup the root partition (/)?
+----------------------------------------
+Backing up your entire root partition works just fine, but remember to exclude directories that makes no sense to backup, such as /dev, /proc, /sys and /run.
```
Comment (ThomasWaldmann) [inline docs/faq.rst]: ... that make no sense ...

Also add /tmp to the list?

Comment (fmikker) [inline docs/faq.rst]: Yeah, the phrasing is way off, was a bit tired. I'll rewrite it and add /tmp to the list.

File: docs/faq.rst [hunk 2]
```diff
@@ -353,6 +353,10 @@ How can I restore huge file(s) over a unstable connection?
 If you can not manage to extract the whole big file in one go, you can extract
 all the part files (see above) and manually concatenate them together.
 
+Can i backup my root partition (/) with borg?
+--------------------------------------------
+Backing up your entire root partition works just fine, but remember to exclude directories that makes no sense to backup, such as /dev, /proc, /sys, /tmp and /run.
```
Comment (ThomasWaldmann) [inline docs/faq.rst]: "make" (not "makes")

**Comment by fmikker** [comment]
Yeah, sorry. Does it make sense now?

**Comment by ThomasWaldmann** [comment]
thanks!

**Pull request merged**